lib/solidus_kustomer/client.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module SolidusKustomer
+  class Client
+    class << self
+      def from_config
+        new(api_key: SolidusKustomer.configuration.api_key)
+      end
+    end
+
+    def initialize(api_key:, url: 'https://api.kustomerapp.com/v1/')
+      @api_key = api_key
+      @url = url
+    end
+
+    # Create a new instance of a Kustomer Klass for a customer
+    #
+    # @param kobject_klass [String] the Klass name of the KObject to create
+    # @param for [Spree::User] the user whose id will own the created KObject
```

Comment:
Looks like the API documentation here is out-of-sync with the actual method signature.

Reply:
You're right, earlier I've chosen `for` but I was afraid of mixing the argument with some reserved keyword.

Thanks for catching that!